### ISSUE TITLE
Fill in details about the networking system

### ISSUE THREAD

**Issue by goatchurchprime** [DoESLiverpool/somebody-should#833]
I'm a bit hazy on what the Rpi in the cabinet is doing (apart from no longer running the MQTT broker), so are there some details we could fill in here: https://github.com/DoESLiverpool/wiki/wiki/Digital-infrastructure#networking

It would be good to state exactly how much internet/bandwidth we should be getting in here, (and reference  bandwidth tests that might have been done with a computer plugged into a cable).  Maybe that funny rpi server (which is running some program I'd like to know the name of) has a feature where it can regularly check and record the internet speed every morning to see if things are improving or deteriorating.

I hit a wall trying to set up the Sonoff-Tasmota (competitor to espurna), which requires me to fill in various things on this file:
https://github.com/arendst/Sonoff-Tasmota/blob/development/sonoff/user_config_override_sample.h

```
#define WIFI_IP_ADDRESS        "0.0.0.0"         // [IpAddress1] Set to 0.0.0.0 for using DHCP or IP address
#define WIFI_GATEWAY           "192.168.2.254"   // [IpAddress2] If not using DHCP set Gateway IP address
#define WIFI_SUBNETMASK        "255.255.255.0"   // [IpAddress3] If not using DHCP set Network mask
#define WIFI_DNS               "192.168.2.27"    // [IpAddress4] If not using DHCP set DNS IP address (might be equal to WIFI_GATEWAY)
```
I don't know what these numbers do, but their default numbers look invalid for our stuff.  We probably need to know what they should be set to here if doing an IoT stuff.

**Comment by ajlennon** [comment]
I think its @johnmckerrell you need to add here. As a great man once said "I know nooooothing..."

**Comment by ajlennon** [comment]
(although those default numbers look fine for DHCP to me. DHCP assigns IP address, gateway, subnet and DNS. With 0.0.0.0 on the top the other numbers will be ignored. Is my guess)

**Comment by goatchurchprime** [comment]
The Sonoff POW2 that was wired to the far-too-long extension lead appears bricked now too.  
It started acting flakey when I flashed Sonoff-Tasmota onto it.  Then I attempted to reflash Espurna and kept getting errors like:

```
esptool.py --port /dev/ttyUSB4 --baud 460800 write_flash --flash_size=detect -fm dio 0 esp8266-20180511-v1.9.4.bin 
esptool.py v2.0
Connecting...
Detecting chip type... ESP8266
Chip is ESP8266
Uploading stub...
Running stub...
Stub running...
Changing baud rate to 460800
Changed.
Configuring flash size...
Auto-detected Flash size: 4MB
Flash params set to 0x0240
Compressed 604872 bytes to 394893...
Writing at 0x00018000... (28 %)
A fatal error occurred: Invalid head of packet (b'\x01')
```

Even when you get something uploaded, it's unresponsive.  

I've checked the USB-UART bridge (the BAITE B75937 one in the components cabinet) and it's reading 3.4V, so shouldn't have blown it, but I'm not going to use it again, just in case.  There are reports of things working by changing your flasher.  https://github.com/arendst/Sonoff-Tasmota/issues/1027  

Sometimes there are bad solder joints that can be fixed.  But this is two in a row, from something that had been working.  I've spent a lot of time on this, and getting quite discouraged.

Basically, the best options are to:
* Use the given Sonoff app first (without flashing)
* Get a different USB-UART bridge
* Convert to Over the air updates
* Experiment only with the cheap S20s

**Comment by ajlennon** [comment]
`A fatal error occurred: Invalid head of packet (b'\x01')`

This implies noisy comms. to me. Try taking the baud rate down or change out the serial.

**Comment by ajlennon** [comment]
Or try a different computer for the programming

**Comment by johnmckerrell** [comment]
Just looking at this, if you're using DHCP you shouldn't need any parameters should you? Our network is actually a 10.0.0.0 network not a 192.168.0.0 network but like I say your parameters seem to suggest you don't need to put stuff in if you're using DHCP.

**Comment by johnmckerrell** [comment]
And actually I now see that the requested link was updated so assuming this issue is really described by its title, I'm going to close it.
